### PR TITLE
RL datasets

### PR DESCRIPTION
**Pull request by Mobius1D**
PR Checklist

- [ ] Update NEWS.md?

Implemented functionality for getting NamedTuple for d4rl, d4rl_atari, d4rl_pybullet with a sample function.

### CODE REVIEW

**Comment by Mobius1D** [comment]
The spacing is kinda bad for the files. Will change it ASAP

File: src/ReinforcementLearningDatasets/.github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: CompatHelper
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/.github/workflows/CompatHelper.yml:1]: The folder ` src/ReinforcementLearningDatasets/.github` can be safely removed.

File: src/ReinforcementLearningDatasets/.github/workflows/TagBot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: TagBot
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/.github/workflows/TagBot.yml:1]: save as above

File: src/ReinforcementLearningDatasets/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/.gitignore:1]: same as above

File: src/ReinforcementLearningDatasets/Project.toml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+uuid = "dd1544ca-2576-438c-a599-ae96278fd687"
+authors = ["Mobius1D"]
+version = "0.1.0"
+
+[deps]
+PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+
+[compat]
+julia = "1"
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/Project.toml:11]: Add compat section of the `deps`

File: src/ReinforcementLearningDatasets/README.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# ReinforcementLearningDatasets
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/README.md:1]: Maybe add some descriptions here?

File: src/ReinforcementLearningDatasets/src/3rd_party/3rd_party_datasets.jl [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+using ArcadeLearningEnvironment:ROM_PATH
+using Random:AbstractRNG
+using ReinforcementLearning: SART, SARTS, BatchSampler
+
+export get_dataset, env_names, install_packages
+
+# supports d4rl, pybullet and atari offline datasets
+# TO-DO: get more info on mujoco envs
+# TO-DO: give warning for getting atari datasets
+function get_dataset(env_name::String, package::String)
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/src/3rd_party/3rd_party_datasets.jl:12]: Could you explain the `env_name` and `package` in the doc string?

File: src/ReinforcementLearningDatasets/src/3rd_party/3rd_party_datasets.jl [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+    
+    data = get_data(env_name, package)
+        
+    if package == "d4rl"
```
Comment (findmyway) [inline src/ReinforcementLearningDatasets/src/3rd_party/3rd_party_datasets.jl:16]: Maybe set the `package` to `"d4rl"` by default?

**Review by findmyway — CHANGES_REQUESTED** [review]

**Comment by findmyway** [comment]
Fix the spell check first?

**Review by findmyway — CHANGES_REQUESTED** [review]
I understand this is a draft version, but we still need to make it workable. First, the project.toml is not updated. Many dependencies are not included, though most of them should be removed later. Second, a simple test is not included yet. You should add the tests in CI.yml file under .github. The current CI failure is another thing. I'll fix it soon.

**Comment by Mobius1D** [comment]
I think we can close this out.

**Pull request closed without merging**